source/linux/Foundational_Components/Power_Management/pm_sw_arch.rst

```diff
@@ -200,6 +195,11 @@ Default constraint is "no constraint", but it can be changed as shown in the exa
 Setting a resume latency constraint impacts the deepest low power mode system can enter.
 The various modes and their latencies are documented here: https://downloads.ti.com/tisci/esd/latest/2_tisci_msgs/pm/lpm.html#tisci-msg-lpm-set-latency-constraint
 
+If a device wants to put a constraint to be not powered-off, it can use the Linux
```

Comment:
to be not powered-off -> to not be powered-off

Reply:
fixed the wording to the suggested